src/core/lib/security/security_connector/ssl_utils.cc

```diff
@@ -220,6 +222,27 @@ int grpc_ssl_cmp_target_name(absl::string_view target_name,
   return overridden_target_name.compare(other_overridden_target_name);
 }
 
+bool isSpiffeID(absl::string_view spiffe_uri) {
```

Comment:
Since this function is local (not exposed in header), you need to use static

`static bool IsSpiffeId(absl::string_view uri) `